### PR TITLE
Fix pull results to ignore actionsets and backfill fetching all runs

### PR DESCRIPTION
**Pull request by norkans7**

### CODE REVIEW

File: ureport/backend/rapidpro.py [hunk 1]
```diff
@@ -349,6 +349,11 @@ def pull_results(self, poll, modified_after, modified_before, progress_callback=
                             ruleset_uuid = temba_step.node
                             category = temba_step.category
                             text = temba_step.text
+                            step_type = temba_step.type
+
+                            if step_type != 'ruleset':
```
Comment (norkans7) [inline ureport/backend/rapidpro.py]: We were adding a result for actionset steps, Added a migrations to remove them

File: ureport/backend/rapidpro.py [hunk 2]
```diff
@@ -308,7 +308,7 @@ def pull_results(self, poll, modified_after, modified_before, progress_callback=
                 start = time.time()
                 print "Start fetching runs for poll #%d on org #%d" % (poll.pk, org.pk)
 
-                poll_runs_query = client.get_runs(flow=poll.flow_uuid, responded=True, after=after, before=now)
+                poll_runs_query = client.get_runs(flow=poll.flow_uuid, after=after, before=now)
```
Comment (norkans7) [inline ureport/backend/rapidpro.py]: Always fetch all runs

File: ureport/public/views.py [hunk 3]
```diff
@@ -195,8 +195,6 @@ def render_to_response(self, context, **kwargs):
 
         results = self.object.get_results(segment=segment)
 
-        results = clean_global_results_data(self.request.org, results, segment)
```
Comment (norkans7) [inline ureport/public/views.py]: this is no longer needed for the global org

**Comment by norkans7** [comment]
@nicpottier please review

**Comment by nicpottier** [comment]
So we need to refetch every poll now?

**Comment by norkans7** [comment]
The migrations will remove the invalid objects with ruleset uuid not in the poll questions we have, those were uuids of actionsets

**Comment by norkans7** [comment]
- Add a fix bug for adding result for actionsets
- Removes all poll results and poll results counters
- Add index to speed up open ended results(this is now fast the table is still empty)

@nicpottier please review

**Comment by nicpottier** [comment]
start it!

**Pull request merged**